src/Files.App/Filesystem/StorageEnumerators/Win32StorageEnumerator.cs

```diff
@@ -47,6 +47,8 @@ public static class Win32StorageEnumerator
 			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
 			bool CalculateFolderSizes = userSettingsService.FoldersSettingsService.CalculateFolderSizes;
 
+			var isGitRepo = GitHelpers.IsRepositoryEx(path, out var repo);
+
```

Comment:
Shouldn't you dispose `repo` at some point?

Reply:
I dont know...

Reply:
In all the examples on their wiki they use `using (var repo = ...)`, so I think we should dispose it

Reply:
Does discarding make any sense? I did so though.

Reply:
Maybe some of the git resources will remain locked until the object is disposed and it will cause an unexpected result when other applications access the repo.

Reply:
Done